src/libraries/System.Text.RegularExpressions/gen/System.Text.RegularExpressions.Generator.csproj

```diff
@@ -22,7 +22,7 @@
 
   <ItemGroup>
     <!-- Common generator support -->
-    <Compile Include="$(CommonPath)Roslyn\GetBestTypeByMetadataName.cs" Link="Common\Roslyn\GetBestTypeByMetadataName.cs" />
+    <Compile Include="$(CommonPath)Roslyn\RoslynExtensions.cs" Link="Common\Roslyn\RoslynExtensions.cs" />
```

Comment:
Looks like this project needs the same change:
https://github.com/dotnet/runtime/blob/f5889ec694ebecdb8688ce3e1b987569caf41348/src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/Microsoft.Extensions.Logging.Generators.targets#L21